examples/schneider_template.html

```diff
@@ -36,11 +36,11 @@ <h2>Test results per Job<h2>
                     <table class="table">
                         <tr>
                             <th scope="row">Job</th>
-                            <td><a href="https://validation.linaro.org/scheduler/job/{{ testrun.job_id }}">Job url</a></td>
+			    <td><a href="https://validation.linaro.org/scheduler/job/{{ testrun.job_id }}">{{ testrun.job_url}}</a></td>
```

Comment:
was this change requested? I think having longer text isn't going to help with legibility of the report.

Reply:
No, it wasn't. But it was in the report we gave Ryan first time. I was thinking this should be the job id from LAVA in stead of 'job url' all over. Not sure what you mean by longer?